providers/yandex/src/airflow/providers/yandex/version_compat.py

```diff
@@ -41,6 +48,8 @@ def get_base_airflow_version_tuple() -> tuple[int, int, int]:
 
 __all__ = [
     "AIRFLOW_V_3_0_PLUS",
+    "AIRFLOW_V_3_1_PLUS",
+    "BaseHook",
```

Comment:
Can you please update the usage in `providers/yandex/src/airflow/providers/yandex/hooks/yandex.py`?

Reply:
done in [35a8cc3](https://github.com/apache/airflow/pull/52963/commits/35a8cc3dfeaf4171bf27e52397d285c3b4d27a25)

Reply:
Thanks!